pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -233,22 +235,14 @@ func (pl *PodTopologySpread) NormalizeScore(ctx context.Context, cycleState *fra
 	}
 
 	for i := range scores {
-		nodeInfo, err := pl.sharedLister.NodeInfos().Get(scores[i].Name)
-		if err != nil {
-			return framework.AsStatus(err)
-		}
-		node := nodeInfo.Node()
-
-		if s.IgnoredNodes.Has(node.Name) {
+		if scores[i].Score == invalidScore {
 			scores[i].Score = 0
 			continue
 		}
```

Comment:
and check it as this,
```
if scores[i].Score == 0 {
    continue
}
```

Reply:
The score coming from `Score` function is basically a counter. If that counter is zero, it should actually get the highest score. See how the value is reversed in the following lines.

Reply:
Got you, thanks for the explanation.